modules/react/collection/lib/useOverflowListModel.tsx

```diff
@@ -7,7 +7,8 @@ export function getHiddenIds(
   containerWidth: number,
   overflowTargetWidth: number,
   itemWidthCache: Record<string, number>,
-  selectedIds: string[] | 'all'
+  selectedIds: string[] | 'all',
+  items?: any[]
```

Comment:
You should use the `Item` interface exported from `./useListBaseModel`:

```suggestion
  items?: Item<any>[]
```

Reply:
Resolved.